### ISSUE TITLE
using-twistedweb howto includes a png which still refers to woven

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3784]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3784|
|Type|defect|
|Created|2009-04-17 14:47:53Z|
|Branch|https://github.com/twisted/twisted/tree/svg-web-howto-3784|

Woven is dead.  The image should be updated to just say "templating system" or something.

<details><summary>Searchable metadata</summary>

```
trac-id__3784 3784
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_svg_web_howto_3784 branches/svg-web-howto-3784
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1239979673000000 1239979673000000
changetime__1260833311000000 1260833311000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

Also, it would be good if this image were kept in revision control in a better format, like svg.  This would make it easier to make future changes to it.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [27728]) Branching to 'svg-web-howto-3784'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

(In [27729]) Update web-process.png and add a .svg version, refs [#3784](https://github.com/twisted/twisted/issues/3784)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Branch is up for review. I created the svg with inkscape for osx, haven't tested it with other editors, but it looked fine in safari..

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Cool.  Thank you.  Looks great.  Please add a news file and merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_8"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [27777]) Merge svg-web-howto-3784: Remove woven from the web-process diagram and add the
.svg source file.

Author: thijs
Reviewer: exarkun
Fixes: [#3784](https://github.com/twisted/twisted/issues/3784)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
